### PR TITLE
V1

### PR DESCRIPTION
**Pull request by samstickland**

### CODE REVIEW

File: hubbado-eventide-service.gemspec [hunk 1]
```diff
@@ -0,0 +1,30 @@
+Gem::Specification.new do |s|
+  s.name = "hubbado-eventide-service"
+  s.version = "0.0.0"
+  s.summary = "Hubbado Eventide Service"
+  s.description = "A gem for starting eventide components"
```
Comment (arepnikov) [inline hubbado-eventide-service.gemspec:5]: with parallelism support

File: lib/eventide_service/component_initiator.rb [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+        tag: :eventide_service
+      )
+
+      # TODO: How to test this?
+      self.class.consumer_registry.each do |consumer, category, options|
+        identifier = options.delete(:identifier)
+        options = options.merge(parallel_consumer_args(identifier: identifier))
+
+        consumer.send(:start, category, **options)
+      end
```
[the remark this answers is not in the document]
Comment (arepnikov) [reply lib/eventide_service/component_initiator.rb:49]: ok, you already have such control: Controls::ComponentInitiator. It probably require at least two Consumers for that test

[the remark this answers is not in the document]
Comment (arepnikov) [reply lib/eventide_service/component_initiator.rb:49]: and I think that you Controls::Consumer should also be fine for that test

**Review by arepnikov — APPROVED** [review]

File: lib/eventide_service/component_initiator.rb (hunk 2 above)
[the remark this answers is not in the document]
Comment (samstickland) [reply lib/eventide_service/component_initiator.rb:49]: Done. Had to add another consumer example as it prevents the same one from being registered twice.

**Pull request merged**